pkg/cgroup/cgroup.go

```diff
@@ -0,0 +1,311 @@
+package cgroup
+
+import (
+	"bufio"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"syscall"
+
+	"github.com/aquasecurity/tracee/pkg/mount"
+)
+
+//
+// Constants
+//
+
+const (
+	procCgroups     = "/proc/cgroups"
+	procFilesystems = "/proc/filesystems"
+
+	CgroupV1FsType = "cgroup"
+	CgroupV2FsType = "cgroup2"
+
+	DefaultCgroupController = "cpuset"
+
+	TmpCgroup       = "/tmp"
```

Comment:
Actually this brings to a good topic... we should use default OS temp directory for temporary files (like these and PID files) AND, perhaps, rename "tmpdir" to something else (it is not a temporary directory, it is an artifacts output directory). No ?

Reply:
Also, if we want to use a "global tmpdir", then we should have global variable to it (instead of having it as part of Tracee object). Or else we will have to pass it as an argument for all packages using it.